data/map.txt

```diff
@@ -4990,6 +5235,130 @@ system Atik
 		distance 1905.05
 		period 270.619
 
+system Atke-babounriuer-rik
+	pos 208.694 45.9351
+	government Xapleaux
+	attributes hot sheltered xapleaux
+	link Ifk-groundissemour
+	link Ink-learoumera-cehikli
+	link Kiteg-hamlior-tock
+	link Vik-benoureal
+	asteroids "small rock" 1 9.75
+	asteroids "medium rock" 1 9.295
+	asteroids "large rock" 2 4.94
+	asteroids "small metal" 17 6.76
+	asteroids "medium metal" 40 3.445
+	asteroids "large metal" 18 9.425
+	minables aluminum 11 9.05446
+	minables lead 3 6.1239
+	minables titanium 3 10.5283
+	haze _menu/haze-full
+	fleet "Xapleaux Small Cargo" 1200
+	fleet "Xapleaux Small Security" 1200
+	fleet "Xapleaux Small Passenger" 1200
+	habitable 589
+	belt 1130
+	trade Clothing 209
+	trade Electronics 643
+	trade Equipment 462
+	trade Food 171
+	trade "Heavy Metals" 1165
+	trade Industrial 554
+	trade "Luxury Goods" 1241
+	trade Medical 708
+	trade Metal 334
+	trade Plastic 395
+	hazard "Core Sheltered Heat" 1
+	hazard "Core Shadow Passing" 9000
+	object
+		sprite star/k8
+		period 10
+	object Atke-babounriuer-taxerik
+		sprite planet/cloud0-b
+		distance 595
+		period 287.25048
+	object
+		sprite planet/ice3
+		distance 1320
+		period 856.64366
+	object
+		sprite planet/gas6-b
+		distance 2816.51
+		period 4259.6788
+		object
+			sprite planet/rock0-b
+			distance 397
+			period 299.01046
+	object
+		sprite planet/browndwarf-l-rouge
```

Comment:
K8 isn't very bright, but it should be enough to outshine this.  And this looks very bad with so many planets between the two.

```suggestion
		sprite planet/browndwarf-l
```